beets/ui/commands.py

```diff
@@ -1022,6 +1022,10 @@ def import_func(lib, opts, args):
     '--pretend', dest='pretend', action='store_true',
     help='just print the files to import'
 )
+import_cmd.parser.add_option(
+    '-m', '--musicbrainzid', dest='musicbrainz_ids', action='append',
+    help='restrict the matching to a single MusicBrainz id'
+)
```

Comment:
This is a tiny nitpick, but these technically don't need to be MBIDs. Other backends can also interpret these IDs (for now, just the Discogs plugin).

Alas, I don't have a great idea for another option name since `-i` and `-I` are both taken. Perhaps `-f` for `--force-id`? Or maybe `-d` as a second-choice contraction of `--id`? Or `-S` for `--search-id`?